### ISSUE TITLE
Ability to use custom ID on insert

### ISSUE THREAD

**Issue by frigus02** [pardom-zz/ActiveAndroid#158]
It would be nice to insert a new entry with a custom id. This is useful for import/export scenarios, where you want to insert many new items on import or restore an old state from an export.

**Comment by jpmermoz** [comment]
Yes! Actually I'm receiving json items from a rest app, and I need to keep the same id's from the remote database. This would be useful.

**Comment by elihart** [comment]
I too would like to see this. It's very important for keeping things consistent with a server. From a quick look at the code it seems you could do this by changing mId in the Model class to be protected instead of private, and then changing the save() method to check if the id exists in the database and updating or inserting accordingly. Then you could set mId in whatever classes you subclass Model from.

That wouldn't be too difficult, but I'm not familiar enough with the rest of the code to know what these changes might break.

**Comment by gustahrodrigues** [comment]
Does this code in the "activeandroid:3.1.0-SNAPSHOT" version ?
